buildSrc/src/main/groovy/com.google.api-ads.java-conventions.gradle

```diff
@@ -35,7 +35,7 @@ repositories {
 }
 
 group = 'com.google.api-ads'
-version = '14.0.1-SNAPSHOT'
+version = '15.0.0'
```

Comment:
Thanks for centralizing this. It should be `15.0.1-SNAPSHOT` to match the pom though, correct?

Reply:
Well the release that just went live is 15.0.0, and this change was intended to be used to release the shadow jar. You are correct though that this should be 15.0.1-SNAPSHOT. I could complete the release and bump again?

Reply:
That SGTM.